presto-hive/src/main/java/com/facebook/presto/hive/util/ConfigurationUtils.java

```diff
@@ -113,7 +113,7 @@ private static void setCompressionProperties(Configuration config, HiveCompressi
             config.unset(FileOutputFormat.COMPRESS_CODEC);
         }
         // For Parquet
-        config.set(ParquetOutputFormat.COMPRESSION, compression.getParquetCompressionCodec().name());
+        compression.getParquetCompressionCodec().ifPresent(parquetCompressionCodec -> config.set(ParquetOutputFormat.COMPRESSION, parquetCompressionCodec.name()));
```

Comment:
nit: parquetCompressionCodec -> codec in the lambda

```
compression.getParquetCompressionCodec().ifPresent(codec -> config.set(ParquetOutputFormat.COMPRESSION, codec.name()));
```

Reply:
@mbasmanova I have already it in lamda?